tests/manifest.yaml

```diff
@@ -10,18 +10,46 @@ test_espresso_total_energy:
   workDir: fixtures/espresso/test-001
   stdoutFile: fixtures/espresso/test-001/pw-scf.out
 
-test_espresso_fermi_energy:
+test_espresso_fermi_energy_pre_v64:
   workDir: fixtures/espresso/test-001
   stdoutFile: fixtures/espresso/test-001/pw-scf.out
 
-test_espresso_nspins:
+test_espresso_nspins_pre_v64:
   workDir: fixtures/espresso/test-001
   stdoutFile: fixtures/espresso/test-001/pw-scf.out
 
-test_espresso_eigenvalues_at_kpoints:
+test_espresso_final_lattice_vectors_pre_v64:
   workDir: fixtures/espresso/test-001
   stdoutFile: fixtures/espresso/test-001/pw-scf.out
 
+test_espresso_eigenvalues_at_kpoints_pre_v64:
+  workDir: fixtures/espresso/test-001
+  stdoutFile: fixtures/espresso/test-001/pw-scf.out
+
+test_espresso_fermi_energy_post_v64:
+  workDir: fixtures/espresso/test-004
+  stdoutFile: fixtures/espresso/test-001/pw-scf.out
+
+test_espresso_nspins_post_v64:
+  workDir: fixtures/espresso/test-004
+  stdoutFile: fixtures/espresso/test-001/pw-scf.out
+
+test_espresso_final_lattice_vectors_post_v64:
+  workDir: fixtures/espresso/test-004
+  stdoutFile: fixtures/espresso/test-001/pw-scf.out
+
+test_espresso_eigenvalues_at_kpoints_post_v64:
+  workDir: fixtures/espresso/test-004
+  stdoutFile: fixtures/espresso/test-001/pw-scf.out
+
+test_espresso_eigenvalues_at_kpoints_lsda_post_v64:
+  workDir: fixtures/espresso/test-005
+  stdoutFile: fixtures/espresso/test-001/pw-scf.out
+
+test_espresso_final_basis_post_v64:
```

Comment:
We should have:

1. Versioning inside the fixtures folder similar to https://mat3ra.atlassian.net/wiki/spaces/PD/pages/2696806427/Fixtures+Repository#Implementation
2. Fixture files (Stdout, XML, or whatever else is needed) for each version supported
3. Tests for each of the property extractors for each of the version fixture files